### PR TITLE
Updates based on review of API documentaton

### PR DESCRIPTION
**Pull request by mmulholla**
Update based in comments in closed PR: https://github.com/redhat-certification/chart-verifier/pull/271

### CODE REVIEW

File: docs/helm-chart-api.md [hunk 1]
```diff
@@ -1,21 +1,29 @@
...
-- The API is available in development release 0.1.0.
+IMPORTANT:
+- The API was first published in chart verifier release 1.8.0
+- The API in release 1.8.0 is not finalized and is subject to change.
+- The API is also available in development release 0.1.0.
 
 ## Overview
 
-The API includes 4 types: Verifier, Report, ReportSummary and Checks:
+The API is written using the go language and was created to enable services to run the chart verifier and obtain a report.
```
[the remark this answers is not in the document]
Comment (mmulholla) [reply docs/helm-chart-api.md:10]: Done.

> @mmulholla Thanks for the changes. I have given some more suggestions. PTAL.
> 
> Also, I see that a good portion of my language suggestions on the use of active voice for descriptions has been ignored. Any particular reason why these language suggestions are ignored or not considered? Please clarify.

I have updated based on those comments, sorry I missed them, hopefully all good now.

File: docs/helm-chart-api.md [hunk 2]
```diff
@@ -137,9 +155,13 @@ func GetChecks() []CheckName
 
 # Example:
 
-Note: Example does not include error checking for clarity.
+Note: 
+- The following example does not include error checking code for clarity purposes.
```
[the remark this answers is not in the document]
Comment (mmulholla) [reply docs/helm-chart-api.md]: @Srivaralakshmi, one thing I have not done is put the api functions into tables. Could not get comfortable with how the table would look and what headings to use. I think it is OK as is for now.

File: docs/helm-chart-api.md [hunk 3]
```diff
@@ -81,19 +97,23 @@ type APIReport interface {
 }
 ```
 
-- NewReport: Used to create a new ```Report```.
+### Description of each function of the APIVerifier interface
+
+- NewReport: Creates a new ```Report```.
   
-- GetContent: Get the report as a string in either json or yaml format. ReportFormat values are defined and available in the report package:
+- GetContent: Gets the report as a string in either json or yaml format. ReportFormat values are defined and available in the report package:
```
Comment (Srivaralakshmi) [inline docs/helm-chart-api.md:104]:
```suggestion
- GetContent: Gets the report as a string in either the JSON or YAML format. ReportFormat values are defined and available in the report package:
```

@mmulholla Please incorporate this feedback for all similar instances.

File: docs/helm-chart-api.md [hunk 4]
```diff
@@ -137,9 +155,13 @@ func GetChecks() []CheckName
 
 # Example:
 
-Note: Example does not include error checking for clarity.
+Note: 
+- The following example does not include error checking code for clarity purposes.
+- For full use of the chart-verifier see:
```
[the remark this answers is not in the document]
Comment (Srivaralakshmi) [reply docs/helm-chart-api.md:172]: @mmulholla yes, please incorporate the suggestion

File: docs/helm-chart-api.md [hunk 5]
```diff
@@ -137,9 +155,13 @@ func GetChecks() []CheckName
 
 # Example:
```
[the remark this answers is not in the document]
Comment (Srivaralakshmi) [reply docs/helm-chart-api.md:165]: @mmulholla Why is this still pending?

File: docs/helm-chart-api.md [hunk 6]
```diff
@@ -150,9 +184,9 @@ import (
 )
 ```
 
-2. Create a verifier, set a profile vendortype of redhat, un-enable the chart testing check, and run verify:
+2. Set a profile.vendortype of redhat, un-enable the chart testing check, and run verify:
```
Comment (Srivaralakshmi) [inline docs/helm-chart-api.md:187]:
```suggestion
2. Set the `profile.vendortype` value to redhat, un-enable the chart testing check, and run verify:
```

**Review by Srivaralakshmi — CHANGES_REQUESTED** [review]
@mmulholla Nice work! Just some minor suggestions. PTAL and incorporate them. Thanks!

File: docs/helm-chart-api.md (hunk 6 above)
Comment (mmulholla) [reply docs/helm-chart-api.md:187]: done.

**Comment by mmulholla** [comment]
@Srivaralakshmi hopefully now all good

**Review by Srivaralakshmi — APPROVED** [review]

**Comment by Srivaralakshmi** [comment]
@mmulholla Thanks for your patience and the changes implemented. Approved!

**Pull request merged**